lib/licensee/content_helper.rb

```diff
@@ -335,9 +335,21 @@ def fields_normalized_set
       @fields_normalized_set ||= fields_normalized.to_set
     end
 
-    def fields_adjusted_length_delta(other)
+    def spdx_alt_segments
+      @spdx_alt_segments ||= begin
+        path = File.expand_path "#{spdx_id}.xml", Licensee::License.spdx_dir
+        raw_xml = File.read(path, encoding: 'utf-8')
```

Comment:
To make things a bit more object-oriented, I'd expect to expose a `xml`-type method on the license itself, which content helper can call, rather than putting the logical to pull the license's XML representation into a helper shared between models.

Reply:
Thanks. I don't want to expose anything yet but I moved the (private) method, you're right it doesn't make sense to have it shared.